### ISSUE TITLE
libcamerasrc doesn't exist after buildin from source.

### ISSUE THREAD

**Issue by bjajoh** [raspberrypi/libcamera#75]
Hi, I installed this version of libcamera like this:
`meson build --buildtype=release -Dpipelines=rpi/vc4 -Dipas=rpi/vc4 -Dv4l2=true -Dgstreamer=enabled -Dtest=false -Dlc-compliance=disabled -Dcam=disabled -Dqcam=disabled -Ddocumentation=disabled -Dpycamera=enabled`

the camera is correctly detected:
`ubuntu@ubuntu:~/libcamera$ libcamera-hello --list-cameras
Available cameras
-----------------
0 : ov5647 [2592x1944 10-bit GBRG] (/base/soc/i2c0mux/i2c@1/ov5647@36)
    Modes: 'SGBRG10_CSI2P' : 640x480 [58.92 fps - (16, 0)/2560x1920 crop]
                             1296x972 [43.25 fps - (0, 0)/2592x1944 crop]
                             1920x1080 [30.62 fps - (348, 434)/1928x1080 crop]
                             2592x1944 [15.63 fps - (0, 0)/2592x1944 crop]`
                             
However gstreamer can't find the libcamerasrc plugin.
`WARNING: erroneous pipeline: no element "libcamerasrc"
`

**Comment by 6by9** [comment]
You haven't given the GStreamer pipeline you're trying to run, and if it contains appsink then there's obviously more than just the pipeline going on. Cut your use case down to the basics for test purposes.
You also appear to have ignored the error message
```
Not enough buffering available for  the processing deadline of 0:00:00.020000000, add enough queues to buffer
```

> All of this works when installing just the binary on Raspbian.

Raspberry Pi OS is the only OS that Raspberry Pi directly support. Support for Ubuntu comes from Canonical.

**Comment by naushir** [comment]
Closing now...
